src/config/mongoose_config_spec.erl

```diff
@@ -172,6 +172,9 @@ general() ->
                  <<"sm_backend">> => #option{type = atom,
                                              validate = {module, ejabberd_sm},
                                              wrap = global_config},
+                 <<"s2s_backend">> => #option{type = atom,
+                                              validate = {module, mongoose_s2s},
+                                              wrap = global_config},
```

Comment:
of course, if the used is weird. It will crash though when init function is called.
But it will pass the validation.
We could check behaviour type - but it is another story.

Reply:
Ok let's leave it for now. It seems like too much effort for little improvement.